airflow/providers/weaviate/hooks/weaviate.py

```diff
@@ -62,29 +63,35 @@ def get_ui_field_behaviour() -> dict[str, Any]:
             },
         }
 
-    def get_client(self) -> weaviate.Client:
+    def get_client(self) -> WeaviateClient:
```

Comment:
Also, should we rename this method to `get_conn`?

Reply:
If we don't care for backward compatibility, I can rename this

Reply:
We can create another function `get_conn` which calls `get_client` and add a deprecation warning in `get_client`?

Reply:
There's already a `get_conn` in the PR. Yeah, good idea to add deprecation